addons/mail/static/src/xml/systray.xml

```diff
@@ -78,6 +78,17 @@
                         <span class="o_preview_name">
                             <t t-esc="activity.name"/>
                         </span>
+                        <div t-if="activity.actions" class="o_mail_activity_action_buttons">
+                            <t t-foreach="activity.actions" t-as="action">
+                                <button type="button"
+                                    t-att-title="action.name"
+                                    t-att-class="'o_mail_activity_action btn btn-link fa ' + action.icon"
+                                    t-att-data-action_xmlid="action.action_xmlid"
+                                    t-att-data-res_model="activity.model"
+                                    t-att-data-model_name="activity.name">
```

Comment:
not sure res_model and model_name are usefull here

Reply:
Correct, it makes sense on `o_mail_preview`, but not here.